src/util/perms.ts

```diff
@@ -1,6 +1,19 @@
+import { settings } from "./../settings";
 import { PermissionFlagsBits } from "discord.js";
 
-export const CIRCLE_PERMS =
-  PermissionFlagsBits.ManageRoles |
-  PermissionFlagsBits.ManageChannels |
-  PermissionFlagsBits.ManageMessages;
+const ENV = "DEV";
+
+const devPerms = {
+  admin: PermissionFlagsBits.Administrator,
+  circle:
+    PermissionFlagsBits.ManageRoles |
+    PermissionFlagsBits.ManageChannels |
+    PermissionFlagsBits.ManageMessages,
+};
+
+const prodPerms = {
+  admin: PermissionFlagsBits.Administrator,
+  circle: BigInt("809997439796117544"),
```

Comment:
Where is this number from? I tried decoding with [this](https://discordapi.com/permissions.html#809997439796117544) but doesn't seem to make sense.

Reply:
It's the id for the circle leader role